pkg/reconciler/route/domains/domains_test.go

```diff
@@ -0,0 +1,170 @@
+package domains
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"github.com/knative/serving/pkg/gc"
+	"github.com/knative/serving/pkg/network"
+	"github.com/knative/serving/pkg/reconciler/route/config"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const TestIngressClass = "ingress-class-foo"
```

Comment:
Is there any need for this constant to be public?

Reply:
Nope, will just inline it with its usage.